src/sentry/options/defaults.py

```diff
@@ -8,13 +8,16 @@
 from __future__ import absolute_import, print_function
 
 
-from sentry.options import register, FLAG_NOSTORE
+from sentry.options import register, FLAG_NOSTORE, FLAG_REQUIRED
 
 
 register('cache.backend', flags=FLAG_NOSTORE)
 register('cache.options', default={}, flags=FLAG_NOSTORE)
-register('system.admin-email')
+register('system.admin-email', flags=FLAG_REQUIRED)
 register('system.databases', default={}, flags=FLAG_NOSTORE)
```

Comment:
In theory, it's possible. But that'd mean, somethign would have to do like:

`options.get('system.databases')['foo'] = 'bar'`

Then subsequent `options.get('system.databases')` would have the mutated value.

Not I don't think this will be an issue now since technically this could happen today with `settings.DATABASES` or anything else. The default isn't shared between keys or anything, so it'd be isolated to this one key. I'm not concerned with this use case.